### ISSUE TITLE
Installation via CMake also brings in 3rd party Dependency

### ISSUE THREAD

**Issue by wilderfield** [wilderfield/priority_map#13]
For some reason Google Benchmark is getting installed when this project is installed.

Need to find a way to prevent that.

Please help if you know the desired CMake magic.

**Comment by wilderfield** [comment]
```
Install the project...
-- Install configuration: ""
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/lib/libbenchmark.a
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/lib/libbenchmark_main.a
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/include/benchmark
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/include/benchmark/export.h
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/include/benchmark/benchmark.h
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/lib/cmake/benchmark/benchmarkConfig.cmake
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/lib/cmake/benchmark/benchmarkConfigVersion.cmake
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/lib/pkgconfig/benchmark.pc
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/lib/cmake/benchmark/benchmarkTargets.cmake
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/lib/cmake/benchmark/benchmarkTargets-noconfig.cmake
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/perf_counters.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/random_interleaving.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/user_guide.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/releasing.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/AssemblyTests.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/dependencies.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/index.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/_config.yml
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/platform_specific_build_instructions.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/assets
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/assets/images
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/assets/images/icon.png
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/assets/images/icon_black.xcf
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/assets/images/icon_black.png
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/assets/images/icon.xcf
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/reducing_variance.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/tools.md
-- Installing: /Users/bryanloz/FrequencyMap/priority_map/priority_map/build/install/share/doc/benchmark/python_bindings.md
```

**Comment by wilderfield** [comment]
Closed by #14
